README.md

```diff
@@ -50,14 +50,11 @@ In addition to the above, the plugin reports the following custom metrics, which
 
 
 # Quality Rules and Profiles
-This plugin integrates 69 quality checks from [Scalastyle](http://www.scalastyle.org/rules-1.0.0.html) and 125 from [Scapegoat](https://github.com/sksamuel/scapegoat/tree/v1.3.5). 40 of the Scalastyle rules are defined without parameters and work out of the box and the remaining 29 are rule templates that allow you to set up custom rules by specifying the parameters yourself. Scapegoat rules don't contain any templates.
+This plugin integrates 69 quality checks from [Scalastyle](http://www.scalastyle.org/rules-1.0.0.html) and 126 from [Scapegoat](https://github.com/sksamuel/scapegoat/tree/v1.3.5). 40 of the Scalastyle rules are defined without parameters and the remaining 29 are rule templates, which allow you to set up custom rules by specifying the parameters yourself. Most of those are are already activated for you with default values recommended by Scalastyle. Scapegoat rules don't contain any templates.
```

Comment:
Update the **Scapegoat** link.
And maybe, add link to [**Scalastyle**](http://www.scalastyle.org/rules-1.0.0.html).

Reply:
Where would you like to add the link to Scalastyle? It's already [here](https://github.com/mwz/sonar-scala/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8R53) and [here](https://github.com/mwz/sonar-scala/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8R60).

Reply:
Nah forget it, I probably was sleepy and didn't see it.